platform/qt/qt.cmake

```diff
@@ -236,24 +279,25 @@ if(MSVC)
         set_target_properties(mbgl-core PROPERTIES COMPILE_PDB_NAME mbgl-cored)
         set_target_properties(qmapboxgl PROPERTIES COMPILE_PDB_NAME QMapboxGLd)
 
-        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-csscolorparser>/mbgl-vendor-csscolorparserd.pdb CONFIGURATIONS "Debug" DESTINATION lib)
-        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-nunicode>/mbgl-vendor-nunicoded.pdb CONFIGURATIONS "Debug" DESTINATION lib)
-        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-parsedate>/mbgl-vendor-parsedated.pdb CONFIGURATIONS "Debug" DESTINATION lib)
+        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-csscolorparser>/mbgl-vendor-csscolorparserd.pdb CONFIGURATIONS "Debug" DESTINATION "${CMAKE_INSTALL_LIBDIR}")
+        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-nunicode>/mbgl-vendor-nunicoded.pdb CONFIGURATIONS "Debug" DESTINATION "${CMAKE_INSTALL_LIBDIR}")
+        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-parsedate>/mbgl-vendor-parsedated.pdb CONFIGURATIONS "Debug" DESTINATION "${CMAKE_INSTALL_LIBDIR}")
 
-        install(FILES $<TARGET_FILE_DIR:mbgl-core>/mbgl-cored.pdb CONFIGURATIONS "Debug" DESTINATION lib)
-        install(FILES $<TARGET_FILE_DIR:qmapboxgl>/QMapboxGLd.pdb CONFIGURATIONS "Debug" DESTINATION lib)
+        install(FILES $<TARGET_FILE_DIR:mbgl-core>/mbgl-cored.pdb CONFIGURATIONS "Debug" DESTINATION "${CMAKE_INSTALL_LIBDIR}")
+        install(FILES $<TARGET_FILE_DIR:qmapboxgl>/QMapboxGLd.pdb CONFIGURATIONS "Debug" DESTINATION "${CMAKE_INSTALL_LIBDIR}")
     else()
-        install(FILES $<TARGET_PDB_FILE:qmapboxgl> CONFIGURATIONS "Debug" DESTINATION bin)
+        install(FILES $<TARGET_PDB_FILE:qmapboxgl> CONFIGURATIONS "Debug" DESTINATION "${CMAKE_INSTALL_BINDIR}")
     endif()
 endif()
 
 install(TARGETS qmapboxgl
-        RUNTIME DESTINATION bin
-        FRAMEWORK DESTINATION lib
-        LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib
-        INCLUDES DESTINATION include
-        PUBLIC_HEADER DESTINATION include
+        EXPORT QMapboxGLTargets
+        RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
```

Comment:
I think that this is not needed to be defined explixitly, if they are set to CMake defaults. The default is set anyway.